sys/Makefile.dep

```diff
@@ -476,6 +476,10 @@ ifneq (,$(filter ieee802154_submac,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter l2util,$(USEMODULE)))
+  USEMODULE += fmt
```

Comment:
Should we maybe make this dependency optional (as in, only use `fmt_byte_hex()` if `fmt` is compiled in, otherwise use a simple local function). It seems weird to include this just for creating strings from addresses, which might not be used with l2util all the time.

Reply:
Unused functions should be thrown out by the linker, so there is nothing to gain from re-implementing `fmt` locally.

Reply:
True. However, you said "should" ;-).

Reply:
ping @benpicco  :)

Reply:
huh? should we really copy the functionality from `fmt` because we don't trust the linker?

Reply:
For me it is less about linking and more about module inter-dependence. But then let's go for using the `fmt` function for now.